### PR TITLE
Chart: describe label.displayFormat

### PR DESCRIPTION
**Pull request by AnnaKondratova**

### CODE REVIEW

File: api-reference/10 UI Components/dxChart/5 Series Types/CommonSeries/label/displayFormat.md [hunk 1]
```diff
@@ -5,7 +5,215 @@ default: undefined
...
-<!-- Description goes here -->
+Specifies the label's text.
+
+---
+
+This property is ignored if labels are [hidden](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/label/#visible). 
+
+Use the parameters of the **label.customizeText** function as placeholders in the **displayFormat** property value.
+
+The example below illustrates how to customize label texts for all and individual series:
```
Comment (albertov05) [inline api-reference/10 UI Components/dxChart/5 Series Types/CommonSeries/label/displayFormat.md]: Why do you say "all _and_ individual series"? Does the code below do both?
Or do you mean "all series or individual series"?

Comment (AnnaKondratova) [reply api-reference/10 UI Components/dxChart/5 Series Types/CommonSeries/label/displayFormat.md]: Yes. The code customizes all labels (common setting for all labels) and an individual label too (it overrides the common setting for the specified label).

Comment (albertov05) [reply api-reference/10 UI Components/dxChart/5 Series Types/CommonSeries/label/displayFormat.md]: Okay, thanks for the clarification. I'd like to change "texts" to "text", and change "all" to "all series" (just to make it more specific).

```suggestion
The example below illustrates how to customize label text for all series and individual series:
```

Comment (AnnaKondratova) [reply api-reference/10 UI Components/dxChart/5 Series Types/CommonSeries/label/displayFormat.md]: Thank you, Albert.

**Pull request merged**